packages/bundle-source/src/tool.js

```diff
@@ -143,6 +143,7 @@ export const makeBundleCache = (wr, cwd, readPowers, opts) => {
         return { relativePath, mtime: mtime.toISOString() };
       }),
     );
+    // eslint-disable-next-line @endo/restrict-comparison-operands
```

Comment:
when we suppress an error there should be a comment with the excuse :) 

but even better don't suppress. wouldn't this pass if `meta` were annotated? I take it `bundleTime` is a string like `mtime.toISOString()`.

Reply:
Added an explanation.